hybridse/src/node/sql_node.cc

```diff
@@ -70,6 +70,7 @@ static absl::flat_hash_map<CmdType, absl::string_view> CreateCmdTypeNamesMap() {
         {CmdType::kCmdShowComponents, "show components"},
         {CmdType::kCmdShowTableStatus, "show table status"},
         {CmdType::kCmdDropFunction, "drop function"},
+        {CmdType::kCmdShowFunctions, "show udf function"},
```

Comment:
add a yaml case in `cases/plan/cmd.yaml` for `SHOW FUNCTIONS`. 
u can test with `ast_node_converter_test` or `planner_v2_test`